services/custom-forms.ts

```diff
@@ -728,3 +728,98 @@ export async function getValueForCustomField(
 
   return fieldValue;
 }
+
+export async function createCustomFormType(name: string): Promise<{
+  id: string;
+}> {
+  const id = uuidv4();
+  const uri = `http://data.lblod.info/id/lmb/forms/custom-types/${id}`;
```

Comment:
can't do this, you will have collisions. You need to add a uuid to the form uri i think

Reply:
(you'll do this when posting to the custom forms instead of separately)